examples/pipelines/controlnet-interior-design/README.md

```diff
@@ -0,0 +1,132 @@
+# ControlNet Interior Design Pipeline
+
+# Table of Contents
+1. [Introduction](#introduction)
+    1. [What is Controlnet?](#what-is-controlnet)
+    2. [Multi-Modal dataset](#multi-modal-dataset)
+    3. [LAION-5B](#laion-5b)
+    4. [Examples](#examples)
+2. [Pipeline Overview](#pipeline-overview)
+    1. [Prompt Generation](#prompt-generation)
+    2. [Image URL Retrieval](#image-url-retrieval)
+    3. [Download Images](#download-images)
+    4. [Add Captions](#add-captions)
+    5. [Add Segmentation Maps](#add-segmentation-maps)
+3. [Install and Run](#install-and-run)
+4. [Reusing this pipeline](#reusing-this-pipeline)
+
+## Introduction
+This example demonstrates an end-to-end fondant pipeline to collect and process data for the training of a [ControlNet](https://github.com/lllyasviel/ControlNet) model, focusing on images related to interior design.
+
+### What is Controlnet?
+
+Controlnet is an image generation model developed by https://arxiv.org/abs/2302.05543 that gives the user more control over the image generation process. It is based on the Stable Diffusion model, which generates images based on a caption and an image. The Controlnet model adds a third input, a conditioning image, that can be used for specifying specific wanted elements in the generated image.
+
+The Controlnet architecture is made so that it can be used for a wide variety of tasks and conditionings, such as segmentation maps, edge maps, scribbles, depth maps and more. A big benefit of Controlnet is that it can be trained with a relatively small dataset, since it reuses a lot of the weights from the Stable Diffusion model. This makes it very accesible for people with limited access to big amounts of compute and data.
+
+Conditioning examples:
+* Semantic segmentation maps
+* Scribbles
+* Depth maps
+* Canny edge maps
+* M-LSD Lines
+* HED Boundaries
+* Human Pose
+* Normal Maps
+* Anime Line Drawings
+* ...
+
+
+Useful links:
+* https://github.com/lllyasviel/ControlNet
+* https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/controlnet
+* https://arxiv.org/abs/2302.05543
```

Comment:
```suggestion
Useful links:

* https://github.com/lllyasviel/ControlNet
* https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/controlnet
* https://arxiv.org/abs/2302.05543
```
It might be you need to include an enter here for this to show appropriately